src/ex6/client/src/components/todo-add/todoAdd.css

```diff
@@ -0,0 +1,48 @@
+input[type=text]{
+    padding-left: 15px;
+    width: 79%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    border: none;
+    background: #f1f1f1;
+}
+  
+input[type=text]:focus{
+    background-color: #ddd;
+    outline: none;
+}
+
+.buttonAdd{
+    background-color: #0095ff;
+    border: 1px solid transparent;
+    border-radius: 3px;
+    box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
+    box-sizing: border-box;
+    color: #fff;
+    cursor: pointer;
+    font-size: 19px;
+    font-weight: 400;
+    line-height: 1.15385;
+    margin: 0;
+    outline: none;
+    padding: 10px;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    user-select: none;
+    -webkit-user-select: none;
+    touch-action: manipulation;
+    vertical-align: baseline;
+    white-space: nowrap;

```

Comment:
that is a lot of CSS properties - did you intend all of them?
extra properties may cause collisions in larger projects